res/css/views/messages/_MessageActionBar.scss

```diff
@@ -75,81 +75,85 @@ limitations under the License.
         cursor: not-allowed;
         opacity: .75;
     }
-}
 
-.mx_MessageActionBar_maskButton::after {
-    content: '';
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100%;
-    width: 100%;
-    mask-size: 18px;
-    mask-repeat: no-repeat;
-    mask-position: center;
-    background-color: $secondary-content;
-}
+    &::after {
```

Comment:
I am not quite sure whether it would be the kind of shortcuts / shorthands to be avoided.

https://github.com/matrix-org/matrix-react-sdk/pull/8131/files#diff-4a24141b03e6fd89fbcdfbc9ca4efe942d8de61d9c753b1edd17e0f88b46c56bR26

Per the example of the yet unofficial guide

> & .mx_MyFoo_avatar { // instead of &_avatar

I believe it rather suggests to avoid cases like these https://github.com/matrix-org/matrix-react-sdk/pull/8131/files#diff-4c20bef27f9bb4d075ed0e6eee2daa41ab2929ad7f7da268a9ec13a7dc33797eL157

Reply:
Also if it would be enforced, a lot of things need to be extracted from nesting. For example, https://github.com/matrix-org/matrix-react-sdk/blob/abd39c61b170c18b9b734dd0b5469ed5a17848af/res/css/views/rooms/_EventTile.scss#L741-L799

They must be converted to 

````
.mx_EventTile[data-shape=ThreadsList]:hover, 
.mx_EventTile[data-shape=ThreadsList]:not([data-layout=bubble]):hover .mx_EventTile_line { 
....
 } 
  
 .mx_EventTile[data-shape=ThreadsList]:not([data-layout=bubble]):hover .mx_EventTile_line { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList]::after, 
.mx_EventTile[data-shape=ThreadsList]::before { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList]::after { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList]::before { 
....
 } 

.mx_EventTile[data-shape=ThreadsList][data-notification]::before {
....
 } 
  
.mx_EventTile[data-shape=ThreadsList][data-notification=total]::before { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList][data-notification=highlight]::before { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList]:last-child { 
.mx_EventTile[data-shape=ThreadsList]:last-child::after { 
....
 } 
  
.mx_EventTile[data-shape=ThreadsList]:first-child { 
....
 } 

````

Honestly, I cannot see any merit of such changes.

Would you please make sure if it would really be expected?

Reply:
Ehm yes. I think you are right :D Sorry for that